ext/net/01_net.js

```diff
@@ -486,6 +487,18 @@ function listen(args) {
   }
 }
 
+function validatePort(maybePort) {
```

Comment:
I think you can make use of `webidl.converters["unsigned short"]` here

Reply:
Hi @mmastrac I did some tests using the function you suggested

```
      const port = webidl.converters['unsigned short'](args.port, undefined, undefined, {
        enforceRange: true
      });

```

Unfortunately it still allows some questionable inputs
With enforce range set to true it will enforce the range per a ushort, but it still allows strings and floats, casting them to integers. 

I can use it for the range check though if that's preferable:

```
function validatePort(maybePort) {
  if (typeof maybePort !== "number" || isNaN(maybePort) ||!Number.isInteger(maybePort) ) {
    throw new TypeError(`Invalid port, expected integer but got: '${maybePort}'`);
  }

  webidl.converters['unsigned short'](maybePort, undefined, undefined, {
    enforceRange: true
  })

  return maybePort;
}
```

However the error message thrown by the converter will be slightly inconsistent (without catching and rethrowing my own)

> Value is outside the accepted range of 0 to 65535, inclusive